npm/webpack-dev-server/src/makeWebpackConfig.ts

```diff
@@ -22,6 +22,9 @@ interface MakeWebpackConfigOptions extends CypressCTOptionsPluginOptions, UserWe
   isOpenMode: boolean
 }
 
+const OsSeparatorRE = RegExp(`\\${path.sep}`, 'g')
```

Comment:
What is the intention with this regex? I think it's invalid, because the first backslash is escaped `\\`, and the second (if Windows) is not `\`.

This is why I said you should do `string.replace(path.sep, '/'`. If:
* POSIX - Replace all forward slashes with forward slashes. Backslashes will stay in place. Nothing wrong
* Windows - Replace all backslashes with forward slashes. All backslashes should directly translate into a POSIX path, with perhaps the exception of the drive letter (`C:\\` -> `C://`). Nothing wrong

Also, if you don't want `/` as a magic value, I believe you can get it with `path.posix.sep` or something like that

Reply:
These regexps are quite hard to read, this might be a good opportunity for some focused unit tests to make it clear what substitution is actually happening.

I am guessing this was tested on an actual windows machine so I have no doubt it works correctly.

Reply:
This regex _will_ have problems on POSIX